src/ContentHub.php

```diff
@@ -12,6 +12,10 @@
 
 class ContentHub extends Client
 {
+    // Override VERSION inherited from GuzzleHttp::ClientInterface
+    const VERSION = '0.6.4';
```

Comment:
This package looks cool. It adds the git commit into the version using the most recent tag. Wouldn't be hard to add it I think. It could take over the decision to set the version information for us. /me likes it...